static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -72,6 +75,44 @@ class Crossword extends React.Component {
                 firstClue.direction
             );
         }
+
+        // Sticky clue
+        const $stickyClueWrapper = $(React.findDOMNode(this.refs.stickyClueWrapper));
+        const $grid = $(React.findDOMNode(this.refs.grid));
+        const $gridViewContainer = $(React.findDOMNode(this.refs.gridViewContainer));
+
+        mediator.on('window:throttledScroll', () => {
+            fastdom.write(() => {
+                // Clear previous state
+                $stickyClueWrapper
+                    .css('top', '')
+                    .css('bottom', '')
+                    .removeClass('is-fixed');
+
+                const { scrollY } = window;
+                const scrollYPastGridViewContainer = scrollY - $gridViewContainer.offset().top;
+
+                if (scrollYPastGridViewContainer >= 0) {
+                    const crosswordOffset = $grid.offset();
+                    const crosswordOffsetBottom = crosswordOffset.top + crosswordOffset.height;
+
+                    if (scrollY > (crosswordOffsetBottom - $stickyClueWrapper.offset().height)) {
+                        $stickyClueWrapper
+                            .css('top', 'auto')
+                            .css('bottom', 0);
+                    } else {
+                        // iOS doesn't support sticky things when the keyboard
+                        // is open, so we use absolute positioning and
+                        // programatically update the value of top
+                        if (detect.isIOS()) {
```

Comment:
don't think we need to check for this on every scroll event, i'd check for it outside of the hander and bind either a ios or non-ios function to scroll there